src/common/reducers/handlers/game/practice.js

```diff
@@ -24,12 +24,18 @@ export function startPractice(state, deck) {
   return state;
 }
 
-export function startSandbox(state) {
+export function startSandbox(state, cardToTest = null) {
   const decks = {
     orange: shuffle(aiDeck).map(card => ({ ...card, id: id() })),
     blue: shuffle(aiDeck).map(card => ({ ...card, id: id() }))
   };
 
+  if (cardToTest) {
+    // If we're entering sandbox mode to test a card, add two copies of it to the top of the orange deck.
```

Comment:
Why two copies?

Reply:
I dunno, I just thought it might make sense to sometimes play a card twice to test it in different scenarios. @jacobnisnevich, thoughts on this?

Reply:
A thought for a future PR is to add a button in Sandbox mode something to the effect of "add test card to hand".